source/index.html

```diff
@@ -1,11 +1,14 @@
 <!DOCTYPE html>
-<html>
+<html lang="en-US">
 
 <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="stylesheet.css">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <script type="text/javascript" src="./assets/instafeed.min.js"></script>
-    <title>Poets Pet Parlor</title>
+    <title>Poet's Pet Parlor</title>
+    <meta name="description" content="a website for Poet's pet parlor. A pet grooming and daycare facility located in Marquette MI.">
```

Comment:
I'd make this a little more conversational, like: `Poet's Pet Parlor is a pet grooming and daycare facility located in Marquette, MI.` … the only place anyone is going to look at this is on the Internet, so no need to say it's a website.